components/script/dom/htmlmediaelement.rs

```diff
@@ -1350,6 +1356,38 @@ impl HTMLMediaElementMethods for HTMLMediaElement {
         self.paused.get()
     }
 
+    /// https://html.spec.whatwg.org/multipage/#dom-media-defaultplaybackrate
+    fn DefaultPlaybackRate(&self) -> Finite<f64> {
+        Finite::wrap(self.defaultPlaybackRate.get())
+    }
+
+    /// https://html.spec.whatwg.org/multipage/#dom-media-defaultplaybackrate
+    fn SetDefaultPlaybackRate(&self, value: Finite<f64>) {
+        if *value != self.defaultPlaybackRate.get() {
+            self.defaultPlaybackRate.set(*value);
+
+            let window = window_from_node(self);
+            let task_source = window.task_manager().media_element_task_source();
+            task_source.queue_simple_event(self.upcast(), atom!("ratechange"), &window);
```

Comment:
We could add a new `queue_ratechange_event` method, so we don't duplicate these lines.

Reply:
Or even better, since this is a pattern that is repeated all over this file, we could have a `fn queue_media_event(&self, event: &str)` method. But it's ok to do this in a follow-up.

Reply:
I chose the first method so as to keep the changes as self-contained as possible. Once this PR is done I'll refactor the whole file using the second method.